DirectEmployers/MyJobs
# Fix compress error.

## What's this PR do?

Removes the need to workaround a missing vendor js bundle.

## Where should the reviewer start?

Not much to this.

## How should this be manually tested?

Remove the `static/bundle` directory from your workspace. Make sure you can still run react apps.

## Any background context you want to provide?

na.
